src/main/java/ch/njol/skript/conditions/base/SkippableCondition.java

```diff
@@ -0,0 +1,26 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions.base;
+
+/**
+ * Interface class for {@link ch.njol.skript.lang.Condition conditions} which can be skipped
+ * <p>Conditions which implement this class can be used in a 'parse if' section</p>
+ */
+public interface SkippableCondition {}
```

Comment:
this interface definitely needs to have some sort of checker method like `canBeChecked`, as a condition may be checkable at parse time under certain circumstances

Reply:
As discussed on discord, I removed this requirement. 
Leaving the class as is for now, to be used more in the future.
Name is still up for discussion.